cluster_traffic_capture/cluster_traffic_capture/gen_haproxy_cfg.py

```diff
@@ -71,7 +71,9 @@ def _get_fe_config_haproxy(haproxy_port: int, mirror: bool) -> str:
     declare capture request len 80000
     declare capture response len 80000
     http-request capture req.body id 0
```

Comment:
Can you 
1/ add a JIRA issue about removing the body from the headers block,
2/ add a comment here explaining the issue with the headers in the body + linking to/mentioning that issue?

Reply:
+1